File: Even.java
import java.lang.*;
import java.util.*;

class Check
{
	private int iNo;
	
	public void Accept()
	{
		Scanner sobj=new Scanner(System.in);
        System.out.println("enter the number");
		
		this.iNo=sobj.nextInt();
	}
	public int CheckEven()
	{
		if((iNo%2)==0)
		{
			return 1;
		}
		else
		{
			return 0;
		}
	}
}
class Even
{
	public static void main(String arg[])
	{ int iRet=0;
		Check cobj=new Check();
		cobj.Accept();
		
		iRet=cobj.CheckEven();
		if(iRet==1)
		{
			System.out.println("given number is even");
		}
		else
		{
		   System.out.println("given number is odd");
		}
	}
	
}

File: Even.c
#include<stdio.h>
#include<stdbool.h>

 bool CheckEven(int iNo)
 {
	 if(iNo<0)
	 {
		 iNo=-iNo;
	 }
	 if((iNo%2)==0)
	 {
		 return true;
	 }
	 else
	 {
		 return false;
	 }
 }
int main()
{
	int iValue=0;
	bool bRet=false;
	
	printf("enter the number\n");
	scanf("%d",&iValue);
	
	bRet=CheckEven(iValue);
	if(bRet==true)
	{
		printf("given number is even\n");
	}
	else
	{
		printf("given number is odd\n");
	}
	
	
	return 0;
}